examples/index.html

```diff
@@ -186,19 +192,36 @@ <h2 style="margin-top:20px">Save</h2>
     </section>
     </div>
   </div>
-
+  <button id="move-up"><i class="fa fa-arrow-circle-o-up"></i></button>
+</body>
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
```

Comment:
I think you have made this branch from your footer branch and not from main. Please remove these changes(footer)